package.json

```diff
@@ -27,23 +27,27 @@
   "matrix-react-parent": "matrix-react-sdk",
   "scripts": {
     "reskindex": "reskindex -h src/header",
+    "build:res": "cpx \"{src/skins/vector/fonts,src/skins/vector/img}/**\" vector/ && cpx \"{res/media,res/vector-icons}/**\" vector/",
+    "build:config": "cpx config.json vector/",
     "build:emojione": "cpx \"node_modules/emojione/assets/svg/*\" vector/emojione/svg/",
     "build:modernizr": "modernizr -c .modernizr.json -d src/vector/modernizr.js",
     "build:css": "mkdirp build && catw \"src/skins/vector/css/**/*.css\" -o build/components.css --no-watch",
     "build:compile": "babel --source-maps -d lib src",
     "build:bundle": "NODE_ENV=production webpack -p --progress",
     "build:bundle:dev": "webpack --optimize-occurence-order --progress",
     "build:electron": "build -lwm",
-    "build": "node scripts/babelcheck.js && npm run build:emojione && npm run build:css && npm run build:bundle",
+    "build": "node scripts/babelcheck.js && npm run build:res && npm run build:config && npm run build:emojione && npm run build:css && npm run build:bundle",
     "build:dev": "npm run build:emojione && npm run build:css && npm run build:bundle:dev",
     "dist": "scripts/package.sh",
+    "start:res": "parallelshell \"cpx -w \\\"{src/skins/vector/fonts,src/skins/vector/img}/**\\\" vector/\" \"cpx \\\"{res/media,res/vector-icons}/**\\\" vector/\"",
```

Comment:
Yeah, if they're the same one, cpx copies it as a whole tree to vector/res[...] and vector/src[...] :(